.eslintrc

```diff
@@ -276,6 +276,7 @@ rules:
       "error",
       {
         "bracketSpacing": false,
-        "singleQuote": true
+        "singleQuote": true,
+        "parser": "babylon"
```

Comment:
Just so I'm clear, this prettierConfig will only run against `*.js` files, correct? I do not believe this `parser` flag is necessary anymore as latest versions of prettier can infer what type of parser to use based on file extension.

I have hit issues with this flag specifically in the past when prettier attempts to format `html` or `json` and we force it into using the `babylon` parser.

Reply:
I'm not 100% sure, but i'm guessing since eslint only runs against js files, that this should be true.

Reply:
@kylecesmat -- I added this here. eslint was producing errors for prettier rule of unspecified parser. And as this is the _eslint_ config and not the prettierc, I'm hoping it should be 🆗